### PR TITLE
Set Volt service to shared and change its name from 'voltService' to 'voltShared'

### PR DESCRIPTION
**Pull request by dschissler**
This is merely a convention change that I hope will ease some of the difficulties that people have had with the Volt service not working correctly when normal and simple views are combined.

### CODE REVIEW

**Comment by dschissler** [comment]
I removed the PHP template engine from the `modules` project because there is custom Volt setup.  The `simple` project has both template engines and this is fine because there is no custom setup there.

**Comment by dschissler** [comment]
The Volt service commit doesn't do much in itself but it will make it easier to explain in documentation, projects and examples when a shared and non-shared Volt service is needed.  This becomes necessary when using `Phalcon\Mvc\View\Simple` views to render emails during a web request.

File: templates/project/modules/Module_web.php [hunk 1]
```diff
@@ -41,8 +41,7 @@ public function registerServices(DiInterface $di)
             $view->setViewsDir(__DIR__ . '/views/');
 
             $view->registerEngines([
-                '.volt'  => 'voltService',
-                '.phtml' => 'Phalcon\Mvc\View\Engine\Php'
```
Comment (sergeyklay) [inline templates/project/modules/Module_web.php]: Return back `.phtml` please

Comment (sergeyklay) [inline templates/project/modules/Module_web.php]: This is te,plate project. Skeleton. Let user to decide what engine to use

Comment (dschissler) [inline templates/project/modules/Module_web.php]: Ok.  Should we be copying over the .phtml resources then?  As it is we only move over the `.volt` files in the build script.

File: templates/project/modules/Module_web.php [hunk 2]
```diff
@@ -41,7 +41,8 @@ public function registerServices(DiInterface $di)
             $view->setViewsDir(__DIR__ . '/views/');
 
             $view->registerEngines([
-                '.volt'  => 'voltShared'
+                '.volt'  => 'voltShared',
+                '.phtml' => 'Phalcon\Mvc\View\Engine\Php'
```
Comment (sergeyklay) [inline templates/project/modules/Module_web.php]: How about? :)

```php
use Phalcon\Mvc\View\Engine\Php as PhpEngine

// ...

$view->registerEngines([
    '.volt'  => 'voltShared',
    '.phtml' => PhpEngine::class
])
```

Comment (dschissler) [inline templates/project/modules/Module_web.php]: I like that.  I'll have to use it in my projects.

File: templates/project/modules/Module_web.php (hunk 1 above)
Comment (sergeyklay) [inline templates/project/modules/Module_web.php]: It seems to me we have to provide just  ability to use this engine. No more

**Comment by sergeyklay** [comment]
Thanks

**Pull request merged**